ElementX/Sources/Services/Client/ClientProxy.swift

```diff
@@ -253,103 +275,91 @@ class ClientProxy: ClientProxyProtocol {
-                .addRange(from: 0, to: 20)
-                .build()
-            
             let allRoomsView = try SlidingSyncViewBuilder()
                 .noTimelineLimit()
-                .requiredState(requiredState: requiredState)
-                .filters(filters: filters)
+                .requiredState(requiredState: slidingSyncRequiredState)
+                .filters(filters: slidingSyncFilters)
                 .name(name: "AllRooms")
                 .syncMode(mode: .growingFullSync)
                 .batchSize(batchSize: 100)
                 .roomLimit(limit: 500)
                 .build()
             
-            let visibleRoomsViewProxy = SlidingSyncViewProxy(slidingSync: slidingSync, slidingSyncView: visibleRoomsView)
-            
             let allRoomsViewProxy = SlidingSyncViewProxy(slidingSync: slidingSync, slidingSyncView: allRoomsView)
             
-            visibleRoomsSummaryProvider = RoomSummaryProvider(slidingSyncViewProxy: visibleRoomsViewProxy,
-                                                              roomMessageFactory: RoomMessageFactory())
-            
             allRoomsSummaryProvider = RoomSummaryProvider(slidingSyncViewProxy: allRoomsViewProxy,
                                                           roomMessageFactory: RoomMessageFactory())
             
-            visibleRoomsViewProxy.visibleRangeUpdatePublisher.sink { [weak self] in
-                self?.restartSync()
-            }
-            .store(in: &cancellables)
-            
-            visibleRoomsViewProxyStateObservationToken = visibleRoomsViewProxy.diffPublisher.sink { [weak self] _ in
-                self?.registerAllRoomSlidingSyncView()
-                self?.visibleRoomsViewProxyStateObservationToken = nil
-            }
-            
-            visibleRoomsSlidingSyncView = visibleRoomsView
             allRoomsSlidingSyncView = allRoomsView
             
         } catch {
-            MXLog.error("Failed building sliding sync views with error: \(error)")
+            MXLog.error("Failed building the all rooms sliding sync view with error: \(error)")
         }
     }
     
+    private lazy var slidingSyncRequiredState: [RequiredState] = {
+        [RequiredState(key: "m.room.avatar", value: ""),
+         RequiredState(key: "m.room.encryption", value: "")]
+    }()
+    
+    private lazy var slidingSyncFilters: SlidingSyncRequestListFilters = {
+        SlidingSyncRequestListFilters(isDm: nil,
+                                      spaces: [],
+                                      isEncrypted: nil,
+                                      isInvite: false,
+                                      isTombstoned: false,
+                                      roomTypes: [],
+                                      notRoomTypes: ["m.space"],
+                                      roomNameLike: nil,
+                                      tags: [],
+                                      notTags: [])
+    }()
```

Comment:
Same here (indentation would need fixing)
```suggestion
    private lazy var slidingSyncFilters = SlidingSyncRequestListFilters(isDm: nil,
                                                                        spaces: [],
                                                                        isEncrypted: nil,
                                                                        isInvite: false,
                                                                        isTombstoned: false,
                                                                        roomTypes: [],
                                                                        notRoomTypes: ["m.space"],
                                                                        roomNameLike: nil,
                                                                        tags: [],
                                                                        notTags: [])
```